.github/workflows/ci.yml

```diff
@@ -49,21 +53,36 @@ jobs:
             experimental: false
             nox-session: unsupported_setup_py
           - python-version: "3.9"
-            os: ubuntu-latest
+            os: ubuntu-20.04
+            experimental: false
+            nox-session: test_brotlipy
```

Comment:
In the interest of reducing our test matrix to the minimum, I think we should only test brotli with one configuration, as brotli has nothing to do with OpenSSL.

Reply:
Done